File: php/1st_week/Praktikum/PersegiPanjang.php
<?php
class PersegiPanjang{
    private $panjang = 0;
    private $lebar = 0;

    public function __construct($panjang = 0, $lebar = 0)
    {
        $this->panjang = $panjang;
        $this->lebar = $lebar;
    }

    public function setPanjang($p){
        $this->panjang = $p;
    }

    public function setLebar($l){
        $this->lebar = $l;
    }

    public function getPanjang(){
        return $this->panjang;
    }

    public function getLebar(){
        return $this->lebar;
    }

    public function keliling(){
        return 2 * ($this->panjang+$this->lebar);
    }

    public function luas(){
        return $this->panjang*$this->lebar;
    }

}

?>

File: java/javadpbo/src/com/sem4dpbo/week1/praktikum/PersegiPanjang.java
package com.sem4dpbo.week1.praktikum;

public class PersegiPanjang {
    private int panjang;
    private int lebar;

//    constructor
    public PersegiPanjang(int p, int l){
        this.panjang = p;
        this.lebar = l;
    }

    public PersegiPanjang(){}

//    method get and set

    public void setPanjang(int panjang) {
        this.panjang = panjang;
    }

    public void setLebar(int lebar) {
        this.lebar = lebar;
    }

    public int getPanjang() {
        return panjang;
    }

    public int getLebar() {
        return lebar;
    }

    public int luas(){
        return panjang * lebar;
    }

    public int keliling(){
        return 2 * (panjang + lebar);
    }
}
